client/src/client.js

```diff
@@ -24,6 +21,15 @@ if (process.version.length != 0) {
   }
 }
 
+const TRACING = false //enable when debugging to see detailed outputs
```

Comment:
This shouldn't be in the code. This setting should be in an .env file, accessed via dotenv, and excluded from the project in gitignore. You should also add an .env.example file that has this one setting in it and it should be set to false. If someone wants to trace, they can create the .env file (in the root of the dist or src folder) with TRACING = true and it should work. It should also not fail if .env file is missing.